### ISSUE TITLE
Manjaro (Arch) Linux: How to Complete Install?

### ISSUE THREAD

**Issue by johntdavis84** [powerline/fonts#363]
Hello,

I'm running Manjaro (Arch-Based)'s latest ARM release.

I've installed `powerline-fonts` from the official repo, but they're not showing up anywhere. Clearly I have not completed the installation, but I'm not sure what I'm doing wrong.

Do I still need to run the shell script provided in the readme? If so, what exactly gets installed when I pull the `powerline-fonts` package from the official repo?

Thanks!

**Comment by ChiefMikeK** [comment]
[Fonts - ArchWiki](https://wiki.archlinux.org/index.php/fonts#Manual_installation )  
did you `fc-cache` ?

**Comment by johntdavis84** [comment]
@ChiefMikeK Thanks for your reply.

I got this working like I wanted.

Turns out everything was working as intended. The `Powerline-fonts` package in the Official Repos for Manjaro just contains a single symbols font that can be substituted for undefined symbols in certain compatible terminal emulators (so you'd set a regular font + a symbol font).

That use-case was a bit too narrow, so I installed the version in the Manjaro AUR that contains the actual set of Powerline fonts. Everything worked automatically as it should.

I'm not really sure why the version with the fonts is not the default, but the ArchWiki entry on Powerline even mentions it, so it's a known thing. I'd never listed out a package to see what it installed before this, so I didn't know how to check until after I ran into trouble...
